internal/config/config.go

```diff
@@ -385,7 +382,13 @@ func peerFromCR(p metallbv1beta2.BGPPeer, passwordSecrets map[string]corev1.Secr
 	if ip == nil {
 		return nil, fmt.Errorf("invalid BGPPeer address %q", p.Spec.Address)
 	}
+
 	holdTime := p.Spec.HoldTime.Duration
+	// DISCUSS1: default value and validation can go away because
+	// it takes place directly in CRD, should we remove them?
+	// DISCUSS2: in RFC the value 0 is valid for holdtime and it means
+	//  BGP session will not expect any Keepalive messages, and it will not
+	//  maintain a session timeout due to inactivity. Do we want to handle it?
```

Comment:
in this case (when 0 has a different meaning than unspecified) we should be using a pointer